dot/rpc/modules/mocks/SyncStateAPI.go

```diff
@@ -0,0 +1,36 @@
+// Code generated by mockery v2.9.4. DO NOT EDIT.
```

Comment:
Do you have `//go:generate` comments for these new mocks?

Reply:
Could you elaborate what you mean by do I have the comments? Not really sure what you mean I just generated the mocks

Reply:
He might have meant this
 
```
//go:generate mockery --name SyncStateAPI --structname SyncStateAPI --case underscore --keeptree

// SyncStateAPI is the interface to interact with sync state.
type SyncStateAPI interface {
	GenSyncSpec(raw bool) (*genesis.Genesis, error)
}
```